jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -149,21 +152,43 @@ public JibContainerBuilder createContainerBuilder(
       Path resourcesOutputDirectory = mainSourceSet.getOutput().getResourcesDir().toPath();
       FileCollection allFiles = mainSourceSet.getRuntimeClasspath().filter(File::exists);
 
-      FileCollection allDependencyFiles =
+      FileCollection projectDependencies =
+          project.files(
+              project
+                  .getConfigurations()
+                  .getByName(JavaPlugin.RUNTIME_CLASSPATH_CONFIGURATION_NAME)
+                  .getResolvedConfiguration()
+                  .getResolvedArtifacts()
+                  .stream()
+                  .filter(
+                      artifact ->
+                          artifact.getId().getComponentIdentifier()
+                              instanceof ProjectComponentIdentifier)
+                  .map(ResolvedArtifact::getFile)
```

Comment:
This is unclear, but should we do `filter(File::exists)`? For what I know, if do the filtering for `allFiles` and `classesOutputDirectories`. (`JavaContainerBuilder` does the check and throws `NoSuchFileException`, which might be a good thing actually. It's unclear to me if this can ever happen.)

Reply:
Yeah I think it's safer to not filter, if we have an issue in the configuration, it should pick it up. `File::exists` will just quietly ignore the error.